### PR TITLE
B/mobile fixes

### PR DESCRIPTION
**Pull request by alavers**
@lemieux @dannytranlx @Mario54 

Some style fixes that make the UI more usable on mobile

* Fix the text input being half cut off
* The first chat messages grow upwards from the bottom rather than starting at the top (helps with iOS, where the keyboard pushes the whole UI upward)

### CODE REVIEW

**Comment by lemieux** [comment]
lgtm :+1:

**Comment by alavers** [comment]
what it actually looks like FYI:

demo

File: src/js/components/conversation.jsx [hunk 1]
```diff
@@ -31,11 +31,8 @@ export class ConversationComponent extends Component {
         let messagesHeight = messages.clientHeight;
         let heightRemaining = conversationHeight - (introHeight + messagesHeight + logoHeight);
 
-        if (heightRemaining > logoHeight) {
-            this.refs.logo.className = 'sk-logo anchor-bottom';
```
Comment (Mario54) [inline src/js/components/conversation.jsx]: Is `anchor-bottom` unused now?

**Comment by Mario54** [comment]
nice :+1:

File: src/js/components/conversation.jsx (hunk 1 above)
Comment (alavers) [inline src/js/components/conversation.jsx]: Good call, the whole `_positionLogo` helper isn't even necessary at all now.

**Pull request merged**